### PR TITLE
Moved routeros sensors to yaml + fixed divisors

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by laf** [comment]
@murrant I'm not sure what the issue is here unless genuinely the move to yaml is breaking something.

Trying to run `./scripts/save-test-data.php -h 399 -o routeros -m sensors` produces:

https://p.libren.ms/view/raw/592c13b7 which seems to contain some bad data. Any ideas without me trawling through the new code?

File: includes/definitions/discovery/routeros.yaml [hunk 1]
```diff
@@ -151,17 +178,24 @@ modules:
...
                 -
                     oid: mtxrHlFanSpeed2
                     num_oid: .1.3.6.1.4.1.14988.1.1.3.18.
                     descr: 'Fan #2'
-                    index: 'mtxrHlFanSpeed2.{{ $index }}'
+                    index: '1'
+        signal:
+            data:
+                -
+                    oid: mtxrWlRtabStrength
```
Comment (murrant) [inline includes/definitions/discovery/routeros.yaml]: This doesn't seem right.  It seems to be wireless client signal strength...  It is indexed by MAC, which caused several issues.

It is also similar to the MSA issue with the string based index.

I would suggest dropping this sensor.

Comment (murrant) [reply includes/definitions/discovery/routeros.yaml]: And don't forget to update the test data.

Comment (laf) [reply includes/definitions/discovery/routeros.yaml]: Updated back to php

**Comment by murrant** [comment]
@laf this looks good to me.  Would you be willing to test the ./save-test-data.php script here?

Capture the data in master, then add here and hopefully everything passes.

**Comment by laf** [comment]
This is all the test data I have :(

**Pull request merged**